api/errors/gameAlreadyStartedError.js

```diff
@@ -0,0 +1,11 @@
+const CustomErrorType = require('./customErrorType');
+
+class GameAlreadyStartedError extends Error {
+  constructor(gameId) {
+    super(`Game ${gameId} has already started!`);
+    this.code = CustomErrorType.GAME_ALREADY_STARTED;
+    this.gameId = gameId;
+  }
```

Comment:
This might be better as a generic `ConflictError` so it's usable in other situations. I'm unsure (conflicted if you will heh) about it because we kindof _want_ to very specifically detect this problem to give it very specific handling, but maybe the context of when a 409 occurs is sufficient to specify it and the error itself can be more generic

Reply:
Swapped it for `ConflictError`. We can see whether this becomes something useful in other situations, but it's more consistent with the other errors being associated with specific status codes and otherwise generic to that status